config/initializers/init_activeadmin_vue.rb

```diff
@@ -0,0 +1,10 @@
+module AdminPageLayoutOverride
+  def build_page(*args)
+    within head do
+      text_node(javascript_packs_with_chunks_tag('admin_application'))
+    end
+    super
+  end
+end
+
+ActiveAdmin::Views::Pages::Base.send :prepend, AdminPageLayoutOverride
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/SendWithMixinArgument: Use prepend AdminPageLayoutOverride instead of send :prepend, AdminPageLayoutOverride.